components/kyma-environment-broker/internal/process/upgrade_kyma_operation_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
why use an archived package? Can you check if you can use "errors" from standard library as it offers similar functionality.

Reply:
Thanks. changed to "errors".